### PR TITLE
admin page feature: edit delivery window options

### PR DESCRIPTION
**Pull request by janiseliang**
image

### CODE REVIEW

**Comment by janiseliang** [comment]
will wait to merge with results from #118

File: pages/api/admin/DeleteDeliveryTime.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+          windowRef.remove()
+          .then(() => {
+            res.status(200);
+            res.json({ message: "success" });
+            return resolve();
+          })
+          .catch(function (error) {
+            res.status(500);
+            res.json({ error: "error deleting order" + orderId, errorstack: error });
```
Comment (justinazhang) [inline pages/api/admin/DeleteDeliveryTime.js:62]: probably never gonna hit this line but just in case probably wanna rephrase this

**Review by justinazhang — CHANGES_REQUESTED** [review]
Yay thanks for doing this :D

File: pages/api/admin/DeleteDeliveryTime.js (hunk 1 above)
Comment (janiseliang) [reply pages/api/admin/DeleteDeliveryTime.js]: great catch, thanks!

**Review by justinazhang — APPROVED** [review]

**Pull request merged**